root/views/layout.sml

```diff
@@ -7,17 +7,17 @@ html
       meta(http-equiv='X-UA-Compatible' content='IE=edge, chrome=1')
       meta(name='description' content='')
       meta(name='author' content='<%= github_username %>')
-      //- uncomment if your site is responsive!
-      //- meta(name="viewport" content="width=device-width, initial-scale=1")
+      meta(name="viewport" content="width=device-width, initial-scale=1")
 
     block(name='title')
       title Title of Your Site
 
     block(name='stylesheets')
       link(rel='stylesheet' href='css/index.css')
 
-  body
-    block(name='content')
+  body(role='document')
+    main(role='main')
```

Comment:
Do you think this is better here, or in the `index` page? I'm on the fence, curious to hear your thoughts on why you placed it here originally.